plaso/parsers/cookie_plugins/ganalytics.py

```diff
@@ -410,29 +410,30 @@ def GetEntries(
       for variable in extra_variables:
         key, _, value = variable.partition('=')
 
-        # Cookies can have a variety of different encodings, usually ASCII or
-        # UTF-8, and values may additionally be URL encoded. urllib only
-        # correctly url-decodes ASCII strings, so we'll convert our string
-        # to ASCII first.
-        try:
-          ascii_value = value.encode('ascii')
-        except UnicodeEncodeError:
-          ascii_value = value.encode('ascii', errors='replace')
-          parser_mediator.ProduceExtractionError(
-              'Cookie contains non 7-bit ASCII characters, which have been '
-              'replaced with a "?".')
-
-        utf_stream = urlparse.unquote(ascii_value)
-
-        try:
-          value_line = utf_stream.decode('utf-8')
-        except UnicodeDecodeError:
-          value_line = utf_stream.decode('utf-8', errors='replace')
-          parser_mediator.ProduceExtractionError(
-              'Cookie value did not decode to Unicode string. Non UTF-8 '
-              'characters have been replaced.')
-
-        extra_attributes[key] = value_line
+        # Urllib2 in Python2 requires a str argument, not unicode. We thus need
```

Comment:
please don't use `str` it is ambiguous I opt to use bytes.

unicode => Unicode

Reply:
These are both the built-in type names from Python 2. I've added some quote marks to make this clear.